lib/controllers/backend/spree/admin/user_passwords_controller.rb

```diff
@@ -9,6 +9,8 @@ class Spree::Admin::UserPasswordsController < Devise::PasswordsController
   helper 'spree/admin/tables'
   layout 'spree/layouts/admin'
 
+  skip_before_action :require_no_authentication, only: [:create]
```

Comment:
Can we require authentication here? We don't want this route to be open to non-admins.

Reply:
👍 The `require_no_authentication` does not open this url to non-admin users, it skips the default behaviour of Devise that only allows to create a password reset request for non-logged in users, what is not practical for admins. They would need to log out to send a password reset token. The actual password reset is handled another controller (`lib/controllers/frontend/spree/user_passwords_controller.rb`)